ql/src/test/org/apache/hadoop/hive/ql/TestTxnCommands2.java

```diff
@@ -1010,7 +952,7 @@ public void testInsertOverwriteWithSelfJoin() throws Exception {
     Assert.assertEquals("Insert overwrite partition failed", stringifyValues(updatedData), rs2);
     //insert overwrite not supported for ACID tables
   }
-  private static void checkCompactionState(CompactionsByState expected, CompactionsByState actual) {
+  private void checkCompactionState(CompactionsByState expected, CompactionsByState actual) {
```

Comment:
nit: new line before

Reply:
fixed

Reply:
Why did we remove `static`? I usually try to keep the methods static, so it shows that the class attributes are not used

Reply:
restored